examples/neutron.rs

```diff
@@ -0,0 +1,187 @@
+use localic_utils::{
+    types::contract::{
+        AuctionStrategy, ChainHaltConfig, MinAmount, PairType, PriceFreshnessStrategy,
+    },
+    ConfigChainBuilder, TestContextBuilder,
+};
+use std::error::Error;
+
+/// Demonstrates using localic-utils for neutron.
+fn main() -> Result<(), Box<dyn Error>> {
+    env_logger::init();
+
+    let neutron = ConfigChainBuilder::default()
+        .with_admin_addr("neutron1kuf2kxwuv2p8k3gnpja7mzf05zvep0cyuy7mxg")
+        .with_chain_id("localneutron-1")
```

Comment:
same, specifying chain id should only be needed when we want to override the hardcoded value from the chain config

Reply:
Added default_neutron, default_osmosis, and default_stride!